scaffolding-go/README.md

```diff
@@ -8,13 +34,15 @@ See https://www.habitat.sh/docs/concepts-scaffolding/ to learn how to get starte
 | Variable | Type | Value | Default |
 | -------- | ---- | ----- | ------- |
 |`scaffolding_go_gopath`| String |_(Optional)_ Value for `GOPATH`|`"$HAB_CACHE_SRC_PATH/$pkg_dirname"`|
```

Comment:
We will need to update the default value for `scaffolding_go_gopath` in the docs too.